src/platform/tasmota/tasmotaUtils.cpp

```diff
@@ -3,24 +3,23 @@
 #include "planetmintgo.h"
 #include "tasmotaUtils.h"
 #include "HttpClientLight.h"
+#include "rddlSDKAbst.h"
+#include "rddlSDKSettings.h"
```

Comment:
Wouldn't it be better if this file was one-sided? Instead of getSetting, tasmotaGetSetting can be used directly in this file

Reply:
valid point! change is under way!